.github/workflows/build-images-beta.yaml

```diff
@@ -18,6 +18,17 @@ permissions:
   id-token: write
 
 jobs:
+  add-inputs-to-env:
+    if: ${{ github.event_name == "workflow_dispatch" }}
+    name: Add Workflow Dispatch Inputs to Env
+    environment: release-beta-images
```

Comment:
I am not sure why the environment variable is set. Is there a reason?

Reply:
I added it in there because the other steps in the `build-images-beta.yaml` workflow used it, but I'm also realizing that I added it into every workflow :facepalm:. Removed.